dockerfiles/ink-waterfall-ci-linux/Dockerfile

```diff
@@ -0,0 +1,60 @@
+ARG VCS_REF=master
+ARG BUILD_DATE
+ARG REGISTRY_PATH=paritytech
+
+FROM ${REGISTRY_PATH}/base-ci:latest
```

Comment:
why it's a separate image? I reckon that you could base it FROM an ink or contracts ci image.